src/server/standard_address_space_part8.cpp

```diff
@@ -699,7 +699,7 @@ namespace OpcUa
         attrs.Type = ObjectID::String;
         attrs.Rank = 1;
         attrs.IsAbstract = false;
-        attrs.Dimensions = std::vector<uint32_t>({0});
+		attrs.Dimensions = { 0 };
```

Comment:
Thanks for the patch. Unfortunately you cannot modify the address_space_partXX files. they are auto-generated from the python script called generate_address_space.py. If you want to make a change you need to modify the python script. In addition that particular change looks strange. Why only that change and why would msvc refuse a  vector constructor and accept an implicit  cast.... . Better propose another constructor syntax